code/index.html

```diff
@@ -1,22 +1,146 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta charset="utf-8"
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Project 2: News-site Economics Today</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
   </head>
+  
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    <!--***Header desktop***-->
+    <header class="header sticky-top">   
+       <nav class="menu">
+          <a href="https://www.technigo.io/">
+            <img class="logo" 
+            src="./logo.svg"
+            alt="Technigo-company-logo"/>  
+         </a>
+         <div class="site-name"><b>ECONOMICS TODAY</b></div>    
```

Comment:
Try to keep the styling to the CSS file, the boldness can be styled with `font-weight`